### ISSUE TITLE
EEG with temporal slowing 

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10829]
**Preferred term label:**

**Synonyms**
EEG with slow-wave activity involving the temporal lobes
EEG with slow activity in temporal lobes

**Definition (free text, please give PubMed ID)**
PMID: 31085562:
Patients with ALE may occasionally have a normal EEG, which does not rule out the diagnosis. Usually, however, EEG shows 
**slow-wave activity** or epileptiform discharges **from the temporal lobes** of patients with ALE — a clue to brain inflammation. In a retrospective analysis of 19 patients with autoimmune encephalitis and seizures, 10 of 16 patients with ictal EEGs (63%) had seizure onset over the temporal lobe region, which closely mirrored the medial temporal lobe abnormalities seen on MRI in three quarters of those studied. Importantly, EEG abnormalities of the temporal lobes without any observed changes on imaging are not sufficient to make a diagnosis of ALE in the absence of a positive antibody, according to the Graus criteria outlined in Box 2. In clinical practice, an EEG that shows **slow-wave activity** or epileptiform discharges **from the temporal lobes** in a patient with possible ALE should raise suspicion of the condition even if the initial MRI is normal; in such cases, repeat MRI may be considered, to look for the interval development of medial temporal lobe abnormalities

**Parent term (use hpo.jax.org/app)**
EEG with focal slow activity HP:0010843

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:623615 Autoimmune limbic encephalitis (ALE)

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 

Adding new term: 
- ID: HP:6001115
- label: EEG with temporal slowing
- parents: [EEG with focal slow activity (HP:0010843)](https://hpo.jax.org/app/browse/term/HP:0010843)
- pmid: PMID:31085562
- def: Slow-wave activity observed in the temporal area of the scalp electro-encephalogram (EEG).
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10829
- synonyms: EEG with slow activity in temporal lobes(NONE)|EEG with slow-wave activity involving the temporal lobes(NONE)
